src/test/groovy/com/synacy/gradprogram/spock/exercise/OrderingServiceSpec.groovy

```diff
@@ -0,0 +1,157 @@
+package com.synacy.gradprogram.spock.exercise
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class OrderingServiceSpec extends Specification {
+
+    def setup(){
+    }
+
+    OrderingService orderingService =  new OrderingService()
+    Item food = ["Burger", 15, ItemType.FOOD]
+    Item food2 = ["Pizza", 5, ItemType.FOOD]
+    Item gadget = ["Cellphone", 10, ItemType.GADGET]
+    Item gadget2 = ["iPad", 5, ItemType.GADGET]
+    Item appliance = ["TV", 20, ItemType.APPLIANCE]
+    Item appliance2 = ["TV", 5, ItemType.APPLIANCE]
+    Item clothing = ["Dress", 10, ItemType.CLOTHING]
+    Item clothing2 = ["Dress", 5, ItemType.CLOTHING]
```

Comment:
Although groovy allows this, I think it would be easier to understand if we instantiate the object using the its constructor.